src/libraries/Common/src/Interop/OSX/System.Native/Interop.Log.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Sys
+    {
+        [DllImport(Libraries.SystemNative, EntryPoint = "SystemNative_Log")]
+        internal static extern bool Log(IntPtr buffer, int count);
```

Comment:
```suggestion
        internal static extern bool Log(void* buffer, int count);
```
We prefer to use actual pointer types for pointers.